lib/src/utils/scope.dart

```diff
@@ -0,0 +1,20 @@
+abstract class Scope {
```

Comment:
It would nice to have some sort of documentation here, and reference to the LinkedIn docs about using scopes